components/centraldashboard/public/components/notebooks-card_test.js

```diff
@@ -1,257 +0,0 @@
-import '@polymer/test-fixture/test-fixture';
```

Comment:
Should you make a test file like this for "notebook-default-card"?

Reply:
Part of another ticket see https://jirab.statcan.ca/browse/BTIS-134